src/chart/treemap/Breadcrumb.ts

```diff
@@ -185,7 +192,18 @@ class Breadcrumb {
                 onclick: curry(onSelect, itemNode)
             });
             (el as ECElement).disableLabelAnimation = true;
-
+            const textContent = el.getTextContent();
+            const stateObj = textContent.ensureState('emphasis');
+            stateObj.style = {
+                fill: emphasisTextStyleModel.getTextColor(),
+                font: emphasisTextStyleModel.getFont(),
+                text
+            };
+            const emphasisState = el.ensureState('emphasis');
+            emphasisState.style = emphasisModel.getModel('itemStyle').getItemStyle();
+            toggleHoverEmphasis(
+                el, 'self', 'global', emphasisModel.get('disabled')
+            );
```

Comment:
I believe it would be great If we could support `focus` & `blur` so that related nodes can be highlighted or downplayed when hovering on the breadcrumb item. Just like this,

![img](https://user-images.githubusercontent.com/26999792/174483399-79c6f200-96c3-4fbc-a437-62f24146560f.gif)

Of course, I know this is optional.